utils.js

```diff
@@ -65,7 +65,7 @@ function sum(iterable) {
 
 function length(iterable) {
     let num = 0;
-    for (let item of iterable) {
+    for (const item of iterable) { // eslint-disable-line no-unused-vars
```

Comment:
Disabled the `no-unused-vars` rule for this line since it was throwing a warning(?) on the unused `item` variable.